### ISSUE TITLE
Karma JS integration broken when "source-map-support" is listed in devDependencies. 

### ISSUE THREAD

**Issue by colmbrady** [neutrinojs/neutrino#1360]
### Bug or issue?

BUG

- What version of Neutrino are you using?

9.0.0-0

- Are you trying to use any presets? If so, which ones, and what versions?

"neutrino": "^9.0.0-0",
 "@neutrinojs/karma": "^9.0.0-0",
  "@neutrinojs/library": "^9.0.0-0",

- Are you using the Yarn client or the npm client? What version?

YARN

- What version of Node.js are you using?

v11.2.0

- What operating system are you using?

Mac

- What did you do?

See repo to reproduce https://github.com/colmbrady/neutrinojs-issue. **README contains a root cause description.**

- What did you expect to happen?

Karma tests to run with no error.

- What actually happened, contrary to your expectations?

Karma tests fail with:

```json
{
    "message": "Uncaught ReferenceError: require is not defined\nat test/index_test.js:1:1\n\nReferenceError: require is not defined\n    at test/index_test.js:1:1",
    "str": "Uncaught ReferenceError: require is not defined\nat test/index_test.js:1:1\n\nReferenceError: require is not defined\n    at test/index_test.js:1:1"
  }
```

**Comment by colmbrady** [comment]
https://github.com/colmbrady/neutrino/blob/v8.3.0/packages/karma/index.js#L61

Seems like the karma integration used to omit plugins previously.

I wonder is the fix for this is to ensure that the plugins are omitted again at this line:

https://github.com/neutrinojs/neutrino/blob/master/packages/karma/index.js#L80

**Comment by edmorley** [comment]
Hi! The testcase repo is missing the `.neutrinorc.js` - could you add it? :-)

**Comment by colmbrady** [comment]
https://github.com/colmbrady/neutrinojs-issue/blob/master/.neutrinorc.js

Woops. Sorry, not sure how that did not get comitted first time.

I think I have noticed that my issues has actually been addressed by this commit.

https://github.com/neutrinojs/neutrino/blob/985e17992ea40618089940ccaba832cce347ec1c/packages/karma/index.js#L59

So perhaps we can just close this?

**Comment by edmorley** [comment]
Ah yes this is a duplicate of #1251, which is fixed by #1319 and due to be released shortly in 9.0.0-rc.1 (#1358).
